### ISSUE TITLE
Redelegating from Dokia Capital to CCN 

### ISSUE THREAD

**Issue by Mickeyt64** [chainapsis/cosmosjs#12]
First tried over an hour ago. 
Retried a couple of times with a message telling me that the first Transaction is yet to go through.

**Comment by Thunnini** [comment]
Could you describe the actual error message?

**Comment by Mickeyt64** [comment]
Sure Champ,

Redelegated from Dokia to CCN a couple of hours ago.

After 30 minutes It still showed Dokia as a Validator so thought maybe it
was my mistake SO

Dokia - Manage - Redelegate - Validator CCN - Set Fee Then first Tx went
through THEN
Message telling me that the original Tranaction has not gone through so
will process new Transaction once this occurs ( note no new transactions
could occur as delegating all Dokia to CCN )
 In short the original Redelegation From Dokia To CCN has Not gone through
after a couple of hours

Cheers Michael

**Comment by Thunnini** [comment]
@Mickeyt64 Sorry for late response.
It seems to be related to the limitation of the redelegation system of the Cosmos.
We didn't explain it intentionally because I thought explaining it to the user could confuse the user.
This [article](https://medium.com/cosmostation/what-you-need-to-know-about-cosmos-atom-redelegation-e45ca7da6fdf) may help you.

**Comment by Mickeyt64** [comment]
Thanks for that information Champion,

Will be very helpful in the future.

Transactions in the last 21days - Deposit in Keplr -
Delelegate -  Unbond part holdings  ( mistake learnt ) - Redelegate -
Redelegate -Then the Redelegation that did not go through.

That would make 0ne Unbond - 2 Successful redelegations & One unsuccessful
Redelegation so something is still not right

Will wait & see what happens I suppose.

Cheers Michael

**Comment by Mickeyt64** [comment]
Hi Champion,
Now see the problem of the 21 day rule of moving a redelegation.
Lesson learnt.
So transaction should go through in 12 days.
Suggestion.
A Info page similar to that found on Puzzle ( secretnodes ) would be
helpful in deciding on who to stake with.

Again, thank you for your time & efforts in helping me out
